pkg/ui/workspaces/cluster-ui/src/store/databaseDetails/databaseDetails.selectors.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import { LocalStorageKeys } from "../localStorage";
+import { SortSetting } from "../../sortedtable";
+import { AppState } from "../reducers";
+import { ViewMode } from "../../databaseDetailsPage";
+import { Filters } from "../../queryFilter";
+import { localStorageSelector } from "../utils/selectors";
+import { KeyedDatabaseDetailsState } from "./databaseDetails.reducer";
+
+export const selectDatabaseDetails = (
```

Comment:
I don't see the functions in this file being used anywhere.

Reply:
These are used in a follow up PR for the connected db details page. I'll remove them from this PR as they're unrelated.